.github/workflows/run-tests.yml

```diff
@@ -25,13 +25,15 @@ jobs:
           - version: '3.10'
             tox-env: py310,py310-mypy,py310-lint,safety
           - version: '3.11'
-            tox-env: py311,py311-mypy,py311-lint,format,safety
+            tox-env: py311,py311-mypy,py311-lint,safety
+          - version: '3.12'
+            tox-env: py312,py312-mypy,format,safety
```

Comment:
Not running Pylint (via Prospector) on Python 3.12 for now since it's crashing. Documented in #739